src/fe-small-advertisements/package.json

```diff
@@ -10,9 +10,13 @@
     "@types/node": "^12.19.14",
     "@types/react": "^16.14.2",
     "@types/react-dom": "^16.9.10",
+    "@types/react-redux": "^7.1.16",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-redux": "^7.2.2",
     "react-scripts": "4.0.1",
+    "redux": "^4.0.5",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
We want this to be a `devDependency`. When we build the project we dont want to have extra packages. This package we use only for development. Can you look into it a little please?

Reply:
Yes, you are right. It makes sense, I actually thought about this when I added it. But then I read that some people actually leave it in production. Anyways, it make more sense what you are saying.